netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -129,6 +134,16 @@ public NettyChannelBuilder flowControlWindow(int flowControlWindow) {
     return this;
   }
 
+  /**
+   * Sets the maximum message size allowed to be received on the channel. If not called,
+   * defaults to 100 MiB.
+   */
+  public NettyChannelBuilder maxMessageSize(int maxMessageSize) {
+    checkArgument(maxMessageSize > 0, "maxMessageSize must be positive");
```

Comment:
is this strictly correct?  A client could feasibly set max message size to 0 and still have meaningful library use.